scripts/kubernetes_node/create.py

```diff
@@ -64,7 +68,7 @@ def execute_command(command):
 if __name__ == '__main__':
 
     # Check if Docker PS works
-    docker = check_command('docker ps')
+    docker = check_command(['sudo', 'docker', 'ps'])
```

Comment:
We shouldn't require passing `sudo`. THe VM Should be created with the agent user in the docker group. Why is this changed needed?